src/wp-admin/includes/user.php

```diff
@@ -1292,7 +1337,7 @@ protected function get_timestamp_as_date( $timestamp ) {
 			return '';
 		}
 
-		$time_diff = current_time( 'timestamp', true ) - $timestamp;
+		$time_diff = time() - $timestamp;
```

Comment:
Is this change for layout or one of the additional changesets?

Reply:
This came from the coding standards change - there must be a ruleset applicable to time functions.

On checking the `current_time()` function it seems passing in 'timestamp' and true simply is a shortcut to `time()` so we could consider leaving this in place.

Reply:
OK